packages/caddy-ingress/v0.2.1+1/package.yaml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://glasskube.dev/schemas/v1/package-manifest.json
+
+name: "caddy-ingress-controller"
```

Comment:
the name must be the same es the directory name. IMO both should be "caddy-ingress"